dist/tools/pr_check/pr_check.sh

```diff
@@ -29,8 +29,15 @@ else
     RIOT_MASTER="master"
 fi
 
+keyword_filter() {
+    grep -i \
+        -e "^    [0-9a-f]\+ .\{0,2\}SQUASH" \
+        -e "^    [0-9a-f]\+ .\{0,2\}FIX" \
+        -e "^    [0-9a-f]\+ .\{0,2\}REMOVEME"
```

Comment:
```suggestion
        -e "^    [0-9a-f]\+ .\{0,2\}REMOVE *ME"
```
?

I've also seen it with spaces in-between.

Reply:
Done and added a commit to verify it's correctly handled.